commit c5542e08ddb93fd03fe4236192d48fe4b02ac3d7 (refs/changes/51/5151/5)
Author: Hans Rosenfeld <hans.rosenfeld@joyent.com>
Date:   2018-12-20T11:04:34+00:00 (10 months ago)
    
    OS-7404 mdb: add KREG_FSBASE
    Reviewed by: John Levon <john.levon@joyent.com>
    Reviewed by: Patrick Mooney <patrick.mooney@joyent.com>
    Approved by: John Levon <john.levon@joyent.com>

diff --git a/usr/src/cmd/mdb/intel/mdb/mdb_kreg.h b/usr/src/cmd/mdb/intel/mdb/mdb_kreg.h
index a3edf864d7..08d65c0cef 100644
--- a/usr/src/cmd/mdb/intel/mdb/mdb_kreg.h
+++ b/usr/src/cmd/mdb/intel/mdb/mdb_kreg.h
@@ -75,6 +75,7 @@ typedef uint32_t kreg_t;
 #define	KREG_ES		KDIREG_ES
 #define	KREG_FS		KDIREG_FS
 #define	KREG_GS		KDIREG_GS
+#define	KREG_FSBASE	KDIREG_FSBASE
 #define	KREG_GSBASE	KDIREG_GSBASE
 #define	KREG_KGSBASE	KDIREG_KGSBASE
 #define	KREG_TRAPNO	KDIREG_TRAPNO
